gromacs/objects/data.py

```diff
@@ -86,7 +86,7 @@ def readTrjInfo(self, protocol, outDir=None):
         outDir = os.path.dirname(self.getSystemFile()) if not outDir else outDir
         infoFile = os.path.abspath(protocol._getPath('logs/run.stderr'))
 
-        command = 'check -f {}'.format(self.getTrajectoryFile())
+        command = 'check -f {}'.format(os.path.abspath(self.getTrajectoryFile()))
```

Comment:
this should be relpath?

Reply:
not here, the problem with the abspaths are in the output sqlites, in order to be able to migrate them to other machines, they should be relative. However, some executions (like this one) do need an abspath (or to configure the cwd and adjust the relpaths, which might be challenging in some cases, so abspath is easier)

Reply:
ok, makes sense. Thanks